backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/User.java

```diff
@@ -11,4 +12,11 @@ public record User(
     String email,
     DocumentationOffice documentationOffice,
     List<String> roles,
-    String initials) {}
+    String initials) {
+
+  @NotNull
```

Comment:
Why do we need this annotation? And why it is a jetbrain annotation? I think with the implementation it can't be null.